bitnami/thanos/templates/querier/tls-server-secret.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if .Values.querier.grpcTLS.server.cert -}}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "thanos.fullname" . }}-querier-tls-server
+  labels: {{- include "thanos.labels" . | nindent 4 }}
+    app.kubernetes.io/component: querier
+type: Opaque
+data:
+  cert.pem: {{ .Values.querier.grpcTLS.server.cert | b64enc | quote }}
+  key.pem: {{ .Values.querier.grpcTLS.server.key | b64enc | quote }}
+  clientca.pem : {{ .Values.querier.grpcTLS.server.clientCA | b64enc | quote }}
```

Comment:
Use `ca` here too to be consistent between the server and client, the mount folder already has the `server` and `client` differentiation.